src/backend/app/db/models.py

```diff
@@ -397,11 +397,6 @@ async def create(
         # Set requesting user to the org owner
         org_in.created_by = user_id
 
-        if not ignore_conflict and cls.one(db, org_in.name):
```

Comment:
Is ignore_conflict used anywhere now?

I forgot why I added it in the first place.
Perhaps it can be removed.

Reply:
Yes, it is used in the init default HOTOSM organisation.

Reply:
It is still there in SQL to create an organization.